lib/models/migration.js

```diff
@@ -45,6 +45,47 @@ module.exports = function(Migration, options) {
   };
 
   /**
+   * Remote Method: Run specific migration by name.
+   *
+   * @param {String} [name] Name of migration script to run.
+   * @param {String} [record] Record the migration runtime to database.
+   * @param {Function} [cb] Callback function.
+   */
+  Migration.migrateByName = function(name, record, cb) {
+    name = name || '';
+    record = record || false;
```

Comment:
I think `record` should be an optional property - you need to add something like this just above this line:

```
    if (typeof cb === 'undefined' && typeof record === 'function') {
      cb = record
      record = false
    }
```